lib/serialize-value.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+var destroyCircular = require('destroy-circular');
+
+// Make a value ready for JSON.stringify() / process.send()
+
+module.exports = function serializeValue(value) {
+	if (typeof value === 'object') {
+		return destroyCircular(value);
+	}
+	if (typeof value === 'function') {
+		// JSON.stringify discards functions
```

Comment:
If a user decides the do `function foo(){}; throw foo`. Terrible idea, but somebody is bound to. Only happens if they actually throw a function (nested functions will be discarded as normal).